easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.9-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'bsddb3'
+version = '6.2.9'
+
+homepage = 'https://pypi.org/project/bsddb3/'
+description = """bsddb3 is a nearly complete Python binding of the
+Oracle/Sleepycat C API for the Database Environment, Database, Cursor,
+Log Cursor, Sequence and Transaction objects."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [PYPI_SOURCE]
```

Comment:
```suggestion
```

This is now a default so, when we download Python packages from pypi, this is not needed.